Language/Haskell/GhcMod/ImportedFrom.hs

```diff
@@ -552,7 +552,10 @@ refineVisibleExports getHaddockInterfaces exports = mapM f exports
             qexports       = qualifiedExports mexports -- e.g. ["base-4.8.2.0:GHC.Base.Just", ...]
         mVisibleExportsMap <- getVisibleExports getHaddockInterfaces pname
 
-        let visibleExportsMap = fromMaybe (throw $ GMEString $ "ImportedFrom: visible exports map is Nothing") mVisibleExportsMap
+        visibleExportsMap <- case mVisibleExportsMap of
```

Comment:
Also `fail` is equivalent to `throwError . GMEString` which is why I suggested you use that before.